_pkgdown.yml

```diff
@@ -0,0 +1,49 @@
+url: https://lintr.r-lib.org
+template:
+  bootstrap: 5
+
+  includes:
+    in_header: |
+      <script defer data-domain="lintr.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
+
+reference:
+- title: Linting
+  contents:
+  - lint_dir
+  - lint_file
+  - lint_package
+  - linters_with_defaults
+  - linters_with_tags
+  - read_settings
+  - exclude
+  - modify_defaults
+  - checkstyle_output
```

Comment:
Not sure where to put this. It's only needed for checkstyle integration, apparently.